nautobot/extras/filters/__init__.py

```diff
@@ -727,6 +729,24 @@ class LocalContextFilterSet(LocalContextModelFilterSetMixin):
     pass
 
 
+#
+# Log Entries
+#
+
+
+class LogEntryFilterSet(BaseFilterSet):
+    q = SearchFilter(
+        filter_predicates={
+            "object_repr": "icontains",
```

Comment:
can/should this also include the `change_message` to match the modeladmin behavior?

Reply:
You mean to search on the `change_message`? Problem is it's stored as an integer and translated to text, so not sure how we'd make it useful without some extras steps.

Reply:
Yeah, since it looks like the modeladmin is set up to search on it above? If not feasible, that's fine, just asking.

Reply:
Oh right, and I'm realizing that we never implemented the search box for the admin integration so that isn't even doing anything. I'll remove it for clarity from the `ModelAdmin`.

Reply:
Actually nevermind, I was mixing up `change_message` and `action_flag`. I'll add `change_message` to search.